src/cache_manager.cc

```diff
@@ -150,27 +150,14 @@ CacheManager::createRequestedAction(const Mgr::ActionParams &params)
     return cmd->profile->creator->create(cmd);
 }
 
-static const CharacterSet &
-MgrFieldChars(const AnyP::ProtocolType &protocol)
-{
-    // Deprecated cache_object:// scheme used '@' to delimit passwords
-    if (protocol == AnyP::PROTO_CACHE_OBJECT) {
-        static const CharacterSet fieldChars = CharacterSet("cache-object-field", "@?#").complement();
-        return fieldChars;
-    }
-
-    static const CharacterSet actionChars = CharacterSet("mgr-field", "?#").complement();
-    return actionChars;
-}
-
 /**
  * define whether the URL is a cache-manager URL and parse the action
  * requested by the user. Checks via CacheManager::ActionProtection() that the
  * item is accessible by the user.
  *
  * Syntax:
  *
- *  scheme "://" authority [ '/squid-internal-mgr' ] path-absolute [ '@' unreserved ] '?' query-string
+ *  scheme "://" authority [ '/squid-internal-mgr' ] path-absolute '?' query-string
```

Comment:
> It is better to document with standard RFC3986 definitions instead of coming up with custom syntax definitions.

Unfortunately, using the standard `path-abempty` does not document the actual syntax. The actual syntax does not use two slashes after the `/squid-internal-mgr` prefix.

We have a choice of either using a rather misleading `/squid-internal-mgr` (without a trailing slash that is actually _required_) OR not using `path-absolute` or `path-abempty`. We can use standard `segment`-based syntax instead, but it will only distract from what we are (or should be) trying to document IMO -- hence my suggestions.

I am not going to veto misleading `/squid-internal-mgr` in the official code or the incorrect double slash in one of the suggestions. I do not insist on any changes to this comment. Whatever you can agree on, use it.